truffle/erc721-example/truffle-config.js

```diff
@@ -1,26 +1,9 @@
-const HDWalletProvider = require('truffle-hdwallet-provider')
+const HDWalletProvider = require('@truffle/hdwallet-provider')
 const NonceTrackerSubprovider = require('web3-provider-engine/subproviders/nonce-tracker')
 const utils = require('web3-utils')
-const MNEMONIC = process.env.MNEMONIC || process.env.MNEMONIC
+const MNEMONIC = process.env.MNEMONIC
 const startIndex = 0
-const numberOfAccounts = 3
-let hdWalletProvider
-
-const setupWallet = (
-    url
-) => {
-    if (!hdWalletProvider) {
-        hdWalletProvider = new HDWalletProvider(
-            MNEMONIC,
-            url,
-            startIndex,
-            numberOfAccounts,
-	    true,
-	)
-        hdWalletProvider.engine.addProvider(new NonceTrackerSubprovider())
```

Comment:
If this is no longer needed - maybe we need to remove it from dependency? But I think we should keep it in case someone will want to use infura.